src/navbar/navbar.js

```diff
@@ -24,7 +24,10 @@ function loadNavbar() {
             const currentPath = window.location.pathname;
             const navLinks = document.querySelectorAll(".nav-link");
             navLinks.forEach((link) => {
-                if (link.getAttribute("href") === currentPath) {
+                if (
```

Comment:
This could use some comments for what it is doing, but we can address this in the cleanup JS issue

Reply:
yep, let me add that